runtime/permissions/lib.rs

```diff
@@ -1233,10 +1420,25 @@ impl Permissions {
       sys: UnaryPermission::allow_all(),
       run: UnaryPermission::allow_all(),
       ffi: UnaryPermission::allow_all(),
+      all: Permissions::new_all(true),
       hrtime: Permissions::new_hrtime(true, false),
     }
   }
 
+  pub fn none(prompt: bool) -> Self {
```

Comment:
Consider adding `none_with_prompt_enabled` and `none_with_prompt_disabled`. Seeing a magic bool passed to `none(false)` made me confused and I had to look up what it means.